manager/types/model.go

```diff
@@ -0,0 +1,46 @@
+/*
+ *     Copyright 2023 The Dragonfly Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package types
+
+type ModelParams struct {
+	ID uint `uri:"id" binding:"required"`
+}
+
+type CreateModelRequest struct {
+	Type        string         `json:"type" binding:"required"`
+	BIO         string         `json:"BIO" binding:"omitempty"`
+	Version     string         `json:"version"  binding:"required"`
+	Evaluation  map[string]any `json:"evaluation" binding:"required"`
```

Comment:
Refer to https://github.com/dragonflyoss/Dragonfly2/blob/main/manager/types/scheduler_cluster.go#L31.